content/applications/productivity/discuss/calls/troubleshooting.rst

```diff
@@ -0,0 +1,118 @@
+===============
+Troubleshooting
+===============
+
+This guide will help you diagnose and resolve issues with Odoo Discuss calls. Follow the checklist
+first, then use the diagnostic tools if needed.
+
+Checklist
+=========
+
+Before exploring advanced troubleshooting options, verify these common issues:
+
+- Is the **browser up to date**?
+- Are there any network-limiting **browser extensions** installed?
+- Is the **Odoo bus** operational?
```

Comment:
How do I know as an average non-technical user?

Reply:
There is the "connection lost" alert that Matthieu added recently